packages/app/src/app/components/Preview/DevTools/elements.ts

```diff
@@ -1,5 +1,12 @@
 import styled, { css } from 'styled-components';
 
+const getCursor = ({ primary, open }) => {
+  if (primary) return 'default';
```

Comment:
Adding curly braces (`{}`) around the body will make this less error-prone.

```suggestion
  if (primary) {
    return 'default';
  }
```